internal/pkg/manifest/lb_web_svc.go

```diff
@@ -206,6 +206,11 @@ func (s LoadBalancedWebService) ApplyEnv(envName string) (*LoadBalancedWebServic
 	if !ok {
 		return &s, nil
 	}
+	envCount := overrideConfig.TaskConfig.Count
+
+	if !envCount.IsEmpty() {
+		s.TaskConfig.Count = envCount // will this mess up the other envs?
```

Comment:
😂 this made me laugh lol 

Can we test this out in a unit test where we apply an `envName` to the override and see if the original count remains intact?

I think this will be safe because the receiver is a concrete type `(s LoadBalancedWebService)` and not `(s *LoadBalancedWebService)` so the original object shouldn't be touched. But we should definitely validate this

Reply:
Do we need to make the same changes for Backend Service as well?

Reply:
gah, yes. keep forgetting that -- would be nice if we could dedupe some of that logic!

Reply:
Anyway, I was just pushing up https://github.com/aws/copilot-cli/pull/2223/commits/577c9b7030ad1c5abe44759c7844f0bee23f90df which should address your first comment!